app/Actions/ElaborateSummary.php

```diff
@@ -37,6 +37,7 @@ public function execute($totalFiles, $changes)
     {
         $summary = new FixReport\ReportSummary(
             $changes,
+            $totalFiles,
```

Comment:
> It was also necessary to pin the php-cs-fixer minor version because they make breaking changes in minor version bumps. Note that the upgrade was necessary in order to get the latest fixers.

BC promise is not covering classes marked as `@internal`. Be careful when playing with internal logic for that reason.

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.0.0/src/Console/Report/FixReport/ReportSummary.php#L20